spec/hydra/works/models/collection_spec.rb

```diff
@@ -15,46 +15,53 @@
 
   describe '#collections' do
     it 'returns empty array when only works are aggregated' do
-      subject.members << generic_work1
-      subject.members << generic_work2
+      subject.ordered_members << generic_work1
```

Comment:
Is it no longer useful to test `members <<` (in addition to `ordered_members <<`) at this layer?